test_edit_weapon fails

def test_edit_weapon(player_wizard, browser): # noqa
        """As a player, I can edit a weapon."""
        print('As a player, I can edit a weapon.')

        weapon_add = weapon.WeaponAddModal(browser)
        weapon_edit = weapon.WeaponEditModal(browser)
        weapon_table = weapon.WeaponTable(browser)
        weapon_tabs = weapon.WeaponModalTabs(browser)
        tabs = Tabs(browser)
        tabs.equipment.click()

        WebDriverWait(browser, 10).until(
            EC.element_to_be_clickable(
                (By.ID, weapon_table.add_id)
            )
        )

        weapon_table.add.click()
        ut.select_from_autocomplete(
            weapon_add,
            'name',
            browser,
            has_search_term=False
        )
        weapon_add.add.click()

        WebDriverWait(browser, 10).until(
            modal_finished_closing(weapon_add.modal_div_id)
        )

        rows = ut.get_table_rows(weapon_table, 'table', values=False)
        rows[0][0].click()

        WebDriverWait(browser, 10).until(
            EC.element_to_be_clickable(
                (By.ID, weapon_tabs.edit_id)
            )
        )

        weapon_tabs.edit.click()

        WebDriverWait(browser, 10).until(
            EC.presence_of_element_located(
                (By.ID, weapon_edit.name_id)
            )
        )

        weapon_edit.name = 'Edit Name'
        weapon_edit.damage = '1d10'
        weapon_edit.magical_modifier = 2
        weapon_edit.to_hit_modifier = 2
        weapon_edit.type_ = 'Melee'
        weapon_edit.handedness = 'One-Handed'
        weapon_edit.proficiency = 'Simple'
        weapon_edit.price = 200
        weapon_edit.currency_denomination = 'GP'
        weapon_edit.weight = 200
        weapon_edit.range_ = 5
        weapon_edit.damage_type = 'Slashing'
        weapon_edit.property_ = 'Versatile'
        weapon_edit.quantity = 2
        weapon_edit.description = 'Edit Description'

        assert weapon_edit.name.get_attribute('value').strip() == 'Edit Name'
        assert weapon_edit.damage.get_attribute('value').strip() == '1d10'
        assert weapon_edit.magical_modifier.get_attribute('value').strip() == '2'
        assert weapon_edit.to_hit_modifier.get_attribute('value').strip() == '2'
        assert weapon_edit.type_.get_attribute('value').strip() == 'Melee'
        assert weapon_edit.handedness.get_attribute('value').strip() == 'One-Handed'
        assert weapon_edit.proficiency.get_attribute('value').strip() == 'Simple'
        assert weapon_edit.price.get_attribute('value').strip() == '200'
        assert weapon_edit.currency_denomination.get_attribute('value').strip() == 'GP'
        assert weapon_edit.weight.get_attribute('value').strip() == '200'
        assert weapon_edit.range_.get_attribute('value').strip() == '5'
        assert weapon_edit.damage_type.get_attribute('value').strip() == 'Slashing'
        assert weapon_edit.property_.get_attribute('value').strip() == 'Versatile'
        assert weapon_edit.quantity.get_attribute('value').strip() == '2'
        assert weapon_edit.description.get_attribute('value').strip() == 'Edit Description'

        weapon_edit.done.click()

        WebDriverWait(browser, 10).until(
            modal_finished_closing(weapon_edit.modal_div_id)
        )

        row = ut.get_table_row(weapon_table, 'table', 1)

>       assert row.weapon.strip() == 'Edit Name  + 2'
E       AssertionError: assert 'Edit Name\xa...          + 2' == 'Edit Name  + 2'
E         + Edit Name  + 2
E         - Edit Name
E         -               + 2

--- a/tests/core/player_tools/test_equipment.py
+++ b/tests/core/player_tools/test_equipment.py
@@ -193,7 +193,9 @@ def test_edit_weapon(player_wizard, browser): # noqa
 
     row = ut.get_table_row(weapon_table, 'table', 1)
 
-    assert row.weapon.strip() == 'Edit Name  + 2'
+    weapon_name_label = ' '.join([string.strip() for string in row.weapon.split()])
+
+    assert weapon_name_label == 'Edit Name + 2'
     assert row.to_hit.strip() == '+ 10'
     assert row.damage.strip() == '1d10'
     assert row.damage_type.strip() == 'Slashing'
@@ -777,6 +779,7 @@ def test_add_armor(player_wizard, browser): # noqa
     assert int(armor_add.price.get_attribute('value').strip()) == stub.price
 
     curr_denomination = stub.currency_denomination
+
     assert armor_add.currency_denomination.get_attribute('value').strip() == curr_denomination
     assert int(armor_add.weight.get_attribute('value').strip()) == stub.weight
     assert int(armor_add.armor_class.get_attribute('value').strip()) == stub.armor_class

